api/lib/opentelemetry/instrumentation/base.rb

```diff
@@ -207,7 +208,7 @@ def install(config = {})
 
         @config = config_options(config)
         instance_exec(@config, &@install_blk)
-        @tracer ||= OpenTelemetry.tracer_provider.tracer(name, version)
+        @tracer = OpenTelemetry.tracer_provider.tracer(name, version)
```

Comment:
This change is made under the assumption that we don't expect users to install/uninstall frequently at runtime.